run.py

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+import sys
+import subprocess
+import threading
+import glob
+
+
+def Str(value):
+    if isinstance(value, list):
+        return " ".join(value)
+    if isinstance(value, basestring):
+        return value
+    return str(value)
+
+
+def Glob(value):
+    ret = glob.glob(value)
+    if(len(ret) < 1):
+        ret = [value]
+    return ret
+
+
+if(str(sys.argv[1]) == str()):
+    print("")
+    print("Enter command arguments with run")
+    print("    run.py install - Install all dependencies for visma")
+    print("    run.py visma - Open visma GUI")
+    print("    run.py test - Run all the tests and generates coverage report")
+    print("    run.py test path/to/test_file.py - Runs all tests and shows coverage for given file")
+    print("    run.py test syntax - Run syntax test using pylama")
+    print("    run.py test modules - Run tests using pytest for all modules")
+    print("    run.py test coverage - After running all the tests, open coverage report")
+    print("    run.py pack - Generate builds for visma package")
+    print("    run.py pack upload - Generate builds and then upload to test.pypi.org")
+    print("    run.py pack final - Generate builds and upload final build to pypi.org")
+    print("    run.py clean - Clean all cache, reports and builds")
+    print("")
+elif (str(sys.argv[1]) == "install"):
+    subprocess.call(["python3 -m pip install -r requirements.txt"], shell=True)
+elif (str(sys.argv[1]) == "visma"):
+    subprocess.call(["python3 main.py"], shell=True)
```

Comment:
Thanks for the PR. When I run this in ubuntu it works fine but when running in windows I get the following error:
```
'"python3 main.py"' is not recognized as an internal or external command,
operable program or batch file.
```
If square brackets around ` ["python3 main.py"]` are removed then it works fine in windows too. Since there is only one element in this list, passing it just as a string will work too. Can you remove the square brackets in other places and test it again?

Reply:
Sure. Let me check that

Reply:
@8hantanu I've made the changes.